spring-xd-shell/src/test/java/org/springframework/xd/shell/command/AbstractSftpSourceTests.java

```diff
@@ -64,7 +64,11 @@ public void setUp() {
 	public void testSftpSource() {
 		String streamName = generateStreamName();
 		CounterSink counter = metrics().newCounterSink(streamName);
-		stream().create(streamName, "sftp --user=%s --password=%s --port=%s --remoteDir=%s --localDir=%s | %s",
+		stream().create(streamName, "sftp --user=%s --password=%s --port=%s --remoteDir=%s --localDir=%s "
+						+ "--allowUnknownKeys=true | %s",
+						// manual test; verify the key was added to the kh file
```

Comment:
I thought it would be useful to leave it as a comment.

I didn't think it was worth the extra effort to set up an automated test for this.